### DISCUSSION TITLE
Launcher States

### DISCUSSION THREAD

**Discussion by HannahStrimpel** [frc5567/2023-Robot#206] [Team Posts]
Cooper and Owen,
   Where are you at with the launcher states? And what can I do to help?

**Comment by CooperTrout** [comment]
I added a Launcher PID, a launcher encoder to use with the PID, and have changed the actions of each state, all of this is based on last year's code. I plan to go through and add comments and double-check things tonight before submitting a pull request. All of the updates I made are on my branch if you would like to look it over.

**Comment by HannahStrimpel** [comment]
Okay
